adobecom/milo
# MWPW-138577: Global footer logging

## Description
This helps sending error logs from the Global Footer in Splunk via LANA.

## Related Issue
Resolves: [MWPW-138577](https://jira.corp.adobe.com/browse/MWPW-138577)

## Testing instructions
1. Change the index value in the Splunk search to `lana_nonprod`
2. In a local environment, manually throw errors where `lanaLog` or `logErrorFor` are used inside `global-footer.js`
3. All the logs sent from your local environment should show in Splunk

## Screenshots
![image](https://github.com/adobecom/milo/assets/146744221/12d4a612-7365-4c8d-a71f-0fbbbfc86381)

## Test URLs
**Acrobat:**
- Before: https://main--dc--adobecom.hlx.page/drafts/rbogos/default?martech=off
- After: https://main--dc--adobecom.hlx.page/drafts/rbogos/default?milolibs=global-footer-logging--milo--robert-bogos&martech=off

**BACOM:**
- Before: https://main--bacom--adobecom.hlx.page/drafts/rbogos/default
- After: https://main--bacom--adobecom.hlx.page/drafts/rbogos/default?milolibs=global-footer-logging--milo--robert-bogos&martech=off

**CC:**
- Before: https://main--cc--adobecom.hlx.page/drafts/rbogos/default?martech=off
- After: https://main--cc--adobecom.hlx.page/drafts/rbogos/default?milolibs=global-footer-logging--milo--robert-bogos&martech=off

**Milo:**
- Before: https://main--milo--robert-bogos.hlx.page/drafts/rbogos/default?martech=off
- After: https://global-footer-logging--milo--robert-bogos.hlx.page/drafts/rbogos/default?martech=off
